cosmos.config.json

```diff
@@ -1,7 +1,8 @@
 {
-  "staticPath": "public",
   "watchDirs": ["src"],
   "webpack": {
-    "configPath": "./cosmos.webpack.config"
-  }
+    "configPath": "react-scripts/config/webpack.config",
+    "overridePath": "cosmos.override.js"
+  },
+  "port": 5001
```

Comment:
so cosmos stuff willa lso be run on yarn start? seems bulkier. but ill give it a go later. just a note